extensions/iidm/include/powsybl/iidm/extensions/iidm/VoltagePerReactivePowerControl.hpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_EXTENSIONS_IIDM_VOLTAGEPERREACTIVEPOWERCONTROL_HPP
+#define POWSYBL_IIDM_EXTENSIONS_IIDM_VOLTAGEPERREACTIVEPOWERCONTROL_HPP
+
+#include <powsybl/iidm/Extension.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+class StaticVarCompensator;
+
+namespace extensions {
+
+namespace iidm {
+
+class VoltagePerReactivePowerControl : public Extension {
+public:  // Extension
+    const std::string& getName() const override;
+
+    const std::type_index& getType() const override;
+
+public:
+    /**
+     * Constructor
+     */
+    VoltagePerReactivePowerControl(StaticVarCompensator& svc, double slope);
+
+    /**
+     * Destructor
+     */
+    ~VoltagePerReactivePowerControl() noexcept override = default;
+
+    /**
+     * Copy constructor
+     */
+    VoltagePerReactivePowerControl(const VoltagePerReactivePowerControl&) = default;
+
+    /**
+     * Move constructor
+     */
+    VoltagePerReactivePowerControl(VoltagePerReactivePowerControl&&) = default;
+
+    /**
+     * Copy assignment operator
+     */
+    VoltagePerReactivePowerControl& operator=(const VoltagePerReactivePowerControl&) = delete;
+
+    /**
+     * Move assignment operator
+     */
+    VoltagePerReactivePowerControl& operator=(VoltagePerReactivePowerControl&&) = delete;
```

Comment:
For other extensions, we don't have all the constructors and operators. It would be weird this extension doesn't have the same behavior than the others. Could you confirm the default/delete constructors/assignment operators are exposed in the same way in the other extensions.
If not, I would prefer you remove these copy constructors/assignment operators, and fix all the extensions in the same time.

Reply:
You're right, they are implcitly declared in other extensions :/
I'll remove declarations in this PR and add them all back in another PR. 

Are every extension movable/copiable/assignable ? 
Or it should be non assignable, like adders : 
```
    XXX(const XXX&) = default;
    XXX(XXX&&) = default;
    XXX& operator=(const XXX&) = delete;
    XXX& operator=(XXX&&) = delete;
```